test/functional/rpc_getdescriptorinfo.py

```diff
@@ -20,9 +20,11 @@ def set_test_params(self):
         self.wallet_names = []
 
     def test_desc(self, desc, isrange, issolvable, hasprivatekeys):
+        expected_normalized_desc_with_checksum = descsum_create(desc)
         info = self.nodes[0].getdescriptorinfo(desc)
         assert_equal(info, self.nodes[0].getdescriptorinfo(descsum_create(desc)))
         assert_equal(info['descriptor'], descsum_create(desc))
+        assert_equal(info['normalizeddescriptor'], expected_normalized_desc_with_checksum)
```

Comment:
This test is never called with a hardened path derivation, so it actually never checks the new logic

Reply:
ah, yes. Missed that. Still "descriptor" and "normalizeddescriptor" are always the same, so it's still not clear why the new field is needed.